clang/test/CodeGenSYCL/sycl_kernel-host.cpp

```diff
@@ -0,0 +1,42 @@
+// RUN: %clang_cc1 -fsycl -fsycl-is-host -triple spir64 -disable-llvm-passes %s -emit-llvm -o -  | FileCheck %s
+// Test that the kernel implementation routine marked with 'sycl_kernel' has the attribute `sycl_kernel` in the generated LLVM IR
+// and that the function object passed to the sycl kernel is marked 'alwaysinline' on the host
```

Comment:
Can you please keep these lines short? i.e. 80 characters.

Reply:
Done, sorry I missed this.